src/server/pegasus_server_impl_init.cpp

```diff
@@ -251,6 +264,22 @@ pegasus_server_impl::pegasus_server_impl(dsn::replication::replica *r)
         tbl_opts.block_cache = _s_block_cache;
     }
 
+    // FLAGS_rocksdb_max_write_rate_megabytes_per_sec <= 0 means close the rate limit.
+    // For more detail arguments see
+    // https://github.com/facebook/rocksdb/blob/master/include/rocksdb/rate_limiter.h#L111-L139
```

Comment:
The referenced 'master' branch code may be changed in the future, you can reference to a certain branch/tag, e.g. https://github.com/facebook/rocksdb/blob/v6.6.4/include/rocksdb/rate_limiter.h#L111-L137